modules/nf-core/scanpy/scrublet/main.nf

```diff
@@ -0,0 +1,47 @@
+process SCANPY_SCRUBLET {
+    tag "${meta.id}"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/cc/cc499354bb89be6429cfc8d8416a340c2327a72732fb6ae33c346feef17b5ffb/data'
+        : 'community.wave.seqera.io/library/python_pyyaml_scanpy_scikit-image:750e7b74b6d036e4'}"
+
+    input:
+    tuple val(meta), path(h5ad)
+    val batch_col
+
+    output:
+    tuple val(meta), path("*.h5ad"), emit: h5ad
+    tuple val(meta), path("*.pkl") , emit: predictions
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    prefix = task.ext.prefix ?: "${meta.id}_scrublet"
+    if ("${h5ad}" == "${prefix}.h5ad") {
+        error("Input and output names are the same, use \"task.ext.prefix\" to disambiguate!")
+    }
+    template('scrublet.py')
+
+    stub:
+    prefix = task.ext.prefix ?: "${meta.id}_scrublet"
+    if ("${h5ad}" == "${prefix}.h5ad") {
+        error("Input and output names are the same, use \"task.ext.prefix\" to disambiguate!")
+    }
+    """
+    export MPLCONFIGDIR=./tmp/mpl
```

Comment:
I'm guessing these are required to not give errors in the version information; if that is the case can you please add a comment to make sure nobody tries to remove them in the future.